api/v1beta1/machine_types.go

```diff
@@ -52,6 +52,14 @@ const (
 	PreTerminateDeleteHookAnnotationPrefix = "pre-terminate.delete.hook.machine.cluster.x-k8s.io"
 )
 
+var (
+	ManagedNodeLabelPrefixes = []string{
```

Comment:
nit:
It would be nice if we define all the entries in this list as a const, both for better readability of our code and for consumers of the API.